pkg/apis/openebs/v1alpha1/blockdevice_types.go

```diff
@@ -39,21 +39,24 @@ type BlockDevice struct {
 
 // DeviceSpec defines the properties and runtime status of a BlockDevice
 type DeviceSpec struct {
-	// NodeAttributes has the details of the node on which BD is attached
-	NodeAttributes NodeAttribute `json:"nodeAttributes"`
 
-	// Path contain devpath (e.g. /dev/sdb)
-	Path string `json:"path"`
+	// AggregateDevice was intended to store the hierachical
+	// information in cases of LVM. However this is currently
+	// not implemented and may need to be re-looked into for
+	// better design.
+	// +optional
+	AggregateDevice string `json:"aggregateDevice,omitempty"`
```

Comment:
This field is deprecated. Can you add into the comments that this field is deprecated?